### DISCUSSION TITLE
backend https

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#495] [Q&A] [answered]
now the frontend in the nginx by https by ssl,but show error when  connect the backend ,
because the the backend support http protocol

so how i can config SSL certificate into the backend,and the backend support https  protocol

**Comment by calebmshafer** [comment, marked as answer]
@paprophet the iModel.js backend is setup to use Express and handles certain configurations for you.  I'd suggest taking a look at this [StackOverflow post](https://stackoverflow.com/questions/11744975/enabling-https-on-express-js) about how to pass the Express server `app` to the Node built-in [https](https://nodejs.org/api/https.html#https_https_createserver_options_requestlistener) methods.

From there you should be able to directly sub-class `IModelJsExpressServer` and set it up so it instantiates a https based server instead of, or in addition to, http.
